acbuild/begin.go

```diff
@@ -49,9 +51,9 @@ func runBegin(cmd *cobra.Command, args []string) (exit int) {
 
 	var err error
 	if len(args) == 0 {
-		err = newACBuild().Begin("", insecure)
+		err = newACBuild().Begin("", insecure, "")
```

Comment:
Potentially. Just making a note that providing "zero values" as positional arguments usually indicates that as the function grows in scope, it's parameter list will get very long, with lots more zero values being provided, so a struct does will in that scenario.

If you're breaking it apart and having callers call the appropriate `Begin*()` function then it's probably unneeded.